README.md

```diff
@@ -6,6 +6,13 @@
 [![CII](https://bestpractices.coreinfrastructure.org/projects/1351/badge)](https://bestpractices.coreinfrastructure.org/projects/1351)
 [![PyPI](https://img.shields.io/pypi/v/tuf)](https://pypi.org/project/tuf/)
 
+----------------------------
+*__IMPORTANT NOTICE:__ A stable 1.0.0 release of the modern implementation only
+is scheduled for January 2022. Please see the [*1.0.0
+announcement*](1.0.0-ANNOUNCEMENT.md) page for more details about the release
+and the deprecation of the legacy implementation, including migration
+instructions.*
```

Comment:
> migration instructions 
It sounds like we have a kind of documentation explaining how to migrate from _current version_ to _1.0.0_.

Not sure if expressing as "migration instructions" fits here. That sounds more when you have a software/product/service and transparent steps to migrate, avoiding issues during this process.

We are deprecating the client.py and repo.py, and we don't have a kind of migration instructions for it. We provide a new repository API and a new powerful Updater library to implement the new repository and client tool.

Reply:
I think the idea here is that we want our current users using the old client and repository API to eventually migrate to the new code.
We already have a working ngclient, but using the new repository API will require more decisions to be made by the users and that's why he focuses on that.

Reply:
@kairoaraujo, I agree that the section at the bottom of the new document, which I'm referencing here, is not a *migration guide* in the strict sense and as you describe it, but it is a set of instructions that should help users to migrate from pre-1.0.0 to 1.0.0. I suggest we leave it for the lack of a better term, unless someone has a good idea.